### DISCUSSION TITLE
I have a bool field and it gaves wierd behaviour.

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2385] [Q&A]
When try to add record using pocketbase dashboard getting error as
![image](https://user-images.githubusercontent.com/14805534/235295773-366bbef2-4b93-434a-aa08-d8161efdd8eb.png)
While adding records from sdk records are getting added and can be listed as well but in dashboard its not showing up. In dashboard while adding record this field always shows as enabled
![image](https://user-images.githubusercontent.com/14805534/235295830-4923233d-5ef1-4ff7-8a36-51ce687994db.png)
But I haven't enabled non falsey for that field also
![image](https://user-images.githubusercontent.com/14805534/235295874-c2e7bd06-4402-4788-a0b4-60e7c696916d.png)

I feel this as wierd behaviour.

**Comment by ganigeorgiev** [comment]
What is the PocketBase version you are using?

The field name (`isNew`) is probably conflicting with the JS SDK isNew method. They were prefixed with `$`, but it is possible that I forgot to replace all occurrences (I'll review them again sometime later today).

**Reply by ganigeorgiev** [reply]
Yes, it is conflicting with the JS SDK helper getter. As a temporary workaround you can use a different field name (eg. `new`).

**Reply by iamdineshkumar** [reply]
Actually I am using flutter sdk, but i get to know issue is due to field name. Then you need to disallow field name which leads to conflict or should have some work around.

**Reply by ganigeorgiev** [reply]
The Admin UI is using the JS SDK under the hood. Renaming the field to something else should "fix" the issue (_a proper fix will be implemented with the next minor v0.15.3 release sometime next week_).

**Reply by iamdineshkumar** [reply]
after changing fild name as work around its working properly...
